src/main/java/me/earth/earthhack/impl/modules/movement/speed/SpeedMode.java

```diff
@@ -16,6 +16,8 @@ public void move(MoveEvent event, Speed module)
         {
             if (mc.player.isElytraFlying()) return;
             if (module.LONG_JUMP.isEnabled()) return;
+            if (!module.Sneak.getValue()
+                    || (!mc.player.isSneaking()))
```

Comment:
Am I blind or is there a return statement missing?

Reply:
I dont think so, if you look at the noWaterInstant setting its not formated like that. I also updated the setting name to "SneakInstant" in the latest commits.

Reply:
> 

If I add a return statement it makes it so when I when the option is off its sneaking and when you turn it on its not sneaking anymore. So makes it contrary to what it should be.

Reply:
Or I can add a return statement and rename the setting to "noSneakInstant" and have it on by default and that would look a little bit better since there is a setting named "noWaterInstant" just above.